quipucords/quipucords/tests_environment.py

```diff
@@ -72,3 +73,8 @@ def test_startup(self, mock_logger):  # pylint: disable=no-self-use
         """Test the startup method."""
         environment.startup()
         mock_logger.assert_called_with(ANY, ANY)
+
+    def test_server_version_default(self):
+        """Test the server version default."""
+        expected = '0.0.0.%s' % environment.commit()
```

Comment:
Should we use the `DEFAULT_VERSION` from `release.py` just in case the default ever changes?

Reply:
yep